tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/JMSNonDestructiveTest.java

```diff
@@ -588,4 +594,82 @@ private void sendLVQTombstone(ConnectionSupplier producerConnectionSupplier, Str
          producer.send(message1, javax.jms.Message.DEFAULT_DELIVERY_MODE, javax.jms.Message.DEFAULT_PRIORITY, tombstoneTimeToLive);
       }
    }
+
+   @Test
+   public void testMultipleLastValuesCore() throws Exception {
+      testMultipleLastValues(CoreConnection);
```

Comment:
Can you instead use CFUtil.createConnectionFactory("AMQP", "tcp://localhost:61616")

or 

CFUtil.createConnectionFactory("CORE", "tcp://localhost:61616")


We should probably remove the Connection supplier. It's a bit simpler to read.